### ISSUE TITLE
Adding multiple navigation items resets addition settings on add.

### ISSUE THREAD

**Issue by IndigoViking** [verbb/navigation#12]
### Description
When adding a navigation item from the sidebar within a navigation, the parent element setting is reset when the item is added. Shouldn't the parent element setting remain the same as you are adding elements especially when you are loading a lot of manual elements. This may not be a problem from entries/categories/assets as you can add several at a time, but manual entry you need to reset it for each entry. Would be faster entry to have this setting remain until changed.

### Steps to reproduce

1. Go into a navigation.
2. Go to manual url and change parent setting.
3. Add item.
4. Setting is reset.

### Additional info

- Plugin version: 1.0.3
- Craft version: Solo 3.0.17.1

**Comment by engram-design** [comment]
That's certainly what's meant to happen, to potentially a bug. Will look into it.

**Comment by The-Indigo-Company** [comment]
Great! Thanks!

**Comment by The-Indigo-Company** [comment]
Also, I'm not sure if this is an issue or the intended way, but when I have children in my navigation structure and I use:
`{% set nodes = craft.navigation.nodes('mainMenu').all() %}`
It will output each node, subnode, and deepnode etc starting at the top of the navitgation and as a child such as:
```
<li>Top Node 1</li>
<ul>
<li>Child Node 1</li>
<li>Child 2</li>
</ul>
<li>Child 1</li>
<li>Child 2</li>
<li>Top Node 2</li>
```
So instead I use
`{% set nodes = craft.navigation.nodes({ handle: 'mainMenu', level: 1 }).all() %}`

Should I file a separate issue or am I doing this wrong?

**Comment by engram-design** [comment]
Yep, this is expected behaviour, and is what regular structures do (which Navigation uses). The biggest reason for not setting the `level` parameter by default is that it's encouraged to use the `{% nav %}` tags to output your items. Setting the default `level` parameter on that doesn't work that well. So - you're doing things the correct way.

Your original issue has been fixed in [1.0.4](https://github.com/verbb/navigation/releases/tag/1.0.4)

**Comment by IndigoViking** [comment]
Thanks! You're the best.
